libs/utils/include/celix_errno.h

```diff
@@ -99,6 +150,10 @@ const char* celix_strerror(celix_status_t status);
 #define CELIX_FRAMEWORK_EXCEPTION (CELIX_START_ERROR + 7)
```

Comment:
I think the error could be replaced with something like:

```
const celix_status_t  CELIX_FRAMEWORK_EXCEPTION = CELIX_ERROR_MAKE(CELIX_FACILITY_SYSTEM,   7);
const celix_status_t  CELIX_FILE_IO_EXCEPTION = CELIX_ERROR_MAKE(CELIX_FACILITY_SYSTEM,             9);
//etc
```

That we make use of the new error code approach and also create an example.